### PR TITLE
1919 Use client to get the latest session from shared keychain

### PR DESCRIPTION
**Pull request by zamorastp**

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Client/Client.swift [hunk 1]
```diff
@@ -233,6 +233,14 @@ public class Client: CustomStringConvertible {
     func destroySession() {
         sessionStorage.remove(forClientId: configuration.clientId)
     }
+    
+    // used only for getting latest session from shared keychain
+    func getLatestSession() -> UserSession? {
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Client/Client.swift:238]: perhaps `getLatestSharedSession`?

File: Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift [hunk 2]
```diff
@@ -30,36 +29,28 @@ public final class SimplifiedLoginManager {
...
+    public init(client: Client,
                 contextProvider: ASWebAuthenticationPresentationContextProviding,
                 env: ClientConfiguration.Environment, // TODO: Currently used to decide language.
                 withMFA: MFAType? = nil,
                 loginHint: String? = nil,
                 extraScopeValues: Set<String> = [],
                 withSSO: Bool = true,
                 completion: @escaping LoginResultHandler) {
-        let sharedKeychainAccessGroup = "\(appIdentifierPrefix).\(SharedKeychainSessionStorageFactory.sharedKeychainGroup)"
-        self.keychainSessionStorage = KeychainSessionStorage(service: Client.keychainServiceName, accessGroup: sharedKeychainAccessGroup)
```
Comment (Daniel-Echegaray) [inline Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginManager.swift:60]: Nice!

**Review by Daniel-Echegaray — APPROVED** [review]
I added a few possible suggestions! But looks good I am approving

File: Sources/AccountSDKIOSWeb/Client/Client.swift (hunk 1 above)
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Client/Client.swift:238]: Done

File: Sources/AccountSDKIOSWeb/Lib/Storage/SessionStorage.swift [hunk 3]
```diff
@@ -23,14 +23,15 @@ internal struct UserTokens: Codable, Equatable, CustomStringConvertible {
...
+    var accessGroup: String? { get }
     func store(_ value: UserSession, accessGroup: String?, completion: @escaping (Result<Void, Error>) -> Void)
     func get(forClientId: String, completion: @escaping (UserSession?) -> Void)  
     func getAll() -> [UserSession]
     func remove(forClientId: String)
     func getLatestSession() -> UserSession?
 }
 
-extension SessionStorage {
+extension SessionStorage {
```
[the remark this answers is not in the document]
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Lib/Storage/SessionStorage.swift:34]: yes :) fixed

**Review by Daniel-Echegaray — APPROVED** [review]
Nice

**Pull request merged**